eq.js

```diff
@@ -38,6 +47,11 @@ const __ = require('./_internal/placeholder')
 const eq = function (left, right) {
   const isLeftResolver = typeof left == 'function',
     isRightResolver = typeof right == 'function'
+
+  if (!isLeftResolver && !isRightResolver) {
```

Comment:
instead of this case here let's instead replace the bottom return `equalBy` with this return. We can also remove the commented out deferred test cases.

Could you also do the same for gt, gte, lt, and lte?